docs/advanced-usage.md

```diff
@@ -143,7 +141,5 @@ To resolve this issue, try the following:
 - Rephrase the prompt and try to generate the playbook steps, see [Prompting tips](#prompting-tips).
 - Verify that your OpenAI account is operational and has enough credit.
```

Comment:
Need to add details about "messagesLimit" settings from key-store

"llmIntegrationData": {
      "openai": {
        "messagesLimit": 20,

Reply:
mentioned in Contents